packages/buffetjs-core/src/components/Enumeration/index.js

```diff
@@ -47,10 +47,14 @@ Enumeration.propTypes = {
   options: PropTypes.arrayOf(
     PropTypes.shape({
       label: PropTypes.string,
-      value: PropTypes.string,
+      value: PropTypes.any,
```

Comment:
Can we also set onOfType propType instead of any ? Like for the value prop below